fiducial_slam/src/map.cpp

```diff
@@ -191,6 +192,7 @@ Map::Map(ros::NodeHandle &nh) : tfBuffer(ros::Duration(30.0)){
     nh.param<std::string>("base_frame", baseFrame, "base_link");
 
     nh.param<float>("tf_publish_interval", tfPublishInterval, 1.0);
+    nh.param<float>("systematic_error", systematic_error, 0.01);
```

Comment:
Can you also add the line to set _useAlexey_

Reply:
OK - so I've done this but I changed it to something a little more meaningful sum_in_quadrature. However the alexy method isn't exactly summing in quadrature - its actually a screwed up version of this. I've decided not to unscrew it up but at least the naming convention is a little better than it was.